CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 
-[Unreleased](https://github.com/puppetlabs/beaker-aws/compare/0.8.1...master)
+## [0.9.1](https://github.com/puppetlabs/beaker-aws/tree/0.9.1) (2021-06-18)
```

Comment:
please don't update the changelog, it's autogenerated for each release based on closed issues and pull requests :)

Reply:
oh, nice. changes dropped down :)